invoices/invoice_expiry_watcher.go

```diff
@@ -139,8 +139,9 @@ func (ew *InvoiceExpiryWatcher) AddInvoices(
 	}
 
 	if len(invoicesWithExpiry) > 0 {
-		log.Debugf("Added %v invoices to the expiry watcher: %v",
-			len(invoicesWithExpiry))
+		log.Debugf("Added %v invoices to the expiry watcher",
```

Comment:
nit: Could be `%d` as we know it's numeric. And I think the logging and error message construction calls are the only exceptions where we don't put the closing parenthesis on a new line.

Reply:
fixed